plugin/trino-base-jdbc/src/test/java/io/trino/plugin/jdbc/BaseJdbcConnectorTest.java

```diff
@@ -1146,4 +1162,120 @@ protected TestView createSleepingView(Duration minimalSleepDuration)
     {
         throw new UnsupportedOperationException();
     }
+
+    @Test
+    public void testDeleteWithBigintEqualityPredicate()
+    {
+        if (!hasBehavior(SUPPORTS_DELETE)) {
+            assertQueryFails("DELETE FROM region WHERE regionkey = 1", "This connector does not support deletes");
```

Comment:
@kokosing In case of connectors where the test infra is shared this can lead to test data being deleted.

Or is the assumption that if the `hasBehavior` is implemented correctly it shouldn't matter since the actual test performs deletions against copies of tables?

Reply:
I'm submitting a PR to change this to follow the approach in `testDeleteWithVarcharEqualityPredicate` where an explicit `TestTable` is created for the negative test case.

Reply:
Thanks!

Reply:
eg https://github.com/trinodb/trino/blob/96be8f29c5b8265f3a3cbccb085a144e79fb270a/testing/trino-testing/src/main/java/io/trino/testing/BaseConnectorTest.java#L700